README.md

```diff
@@ -2,27 +2,39 @@
 
 This Google Apps script creates "busy" events on a calendar based on another.
 
-I am using this to mark the events on my personal calendar as "busy" on my work calendar.
+I am using this to mark the events on my personal calendar as "busy" on my
+work calendar.
 
 ## How it works
 
-It fetches all events from `SOURCE_CALENDAR` between 2 weeks ago and 2
-months from now and events with a specific tag from `TARGET_CALENDAR`,
-for the same interval. Then it compares them to figure out which events
-to insert to and delete from `TARGET_CALENDAR`, and applies the changes
-with the correct tag.
+It fetches all events from `SOURCE_CALENDAR` between 1 week ago and 2 months
+from now and events with a specific tag from `TARGET_CALENDAR`, for the same
+interval. Then it compares them to figure out which events to insert to and
+delete from `TARGET_CALENDAR`, and applies the changes with the correct tag.
 
 I tried to make it hard for the code to mess with the events it is not
 managing. So:
 
 * It only ever modifies `TARGET_CALENDAR`.
-* It only reads `startTime` and `endTime` fields from the events in
-  `SOURCE_CALENDAR`.
+* It only reads the following fields from the events in `SOURCE_CALENDAR`:
+   `startTime`, `endTime`, `allDayEvent`, `myStatus`, and `tag`. (Note, all
+   but the first two may be unavailable depending on sharing settings.)
 * It can only modify/create events with a certain tag (`GCAL_I_AM_BUSY`).
 
-However, there are no tests or real world use (yet). So, use at your
+However, the real world always presents unexpected conditions, so, use at your
 own risk. But do create an issue or send a PR if it does something wrong.
 
+## Bidirectional syncing
+
+It's possible to use this script on both sides of a pair of calendars shared
+to one another to synchronize bidirectionally. In this case the script tries
+to make sure it doesn't sync events it itself has generated. It will look for
+its `GCAL_I_AM_BUSY` tag, but if the foreign calendar is set to share only
+busy/free (start/end times of confirmed events), the tag will not be
+visible. For this reason, it sets the `seconds` field of the start time of
+events it creates to 1 second, and will not sync any events that have a
```

Comment:
"1 second" does make sense, but I feel like "1" is an ubiquitous number that can happen for some other reason (that I cannot think of any, but [Benford's law](https://en.wikipedia.org/wiki/Benford%27s_law) is a thing). So maybe we should go with a weird one like "13 seconds past" or something.

Reply:
Wow, Benford's law is wild! I now changed it to 41.